manifests/client/mount.pp

```diff
@@ -114,4 +117,14 @@
       require  => Nfs::Functions::Mkdir[$mountname]
     }
   }
+
+  if $owner != undef or $group != undef or $mode != undef {
```

Comment:
It shouldn't crash - if any of the values are not set, it should just result in nothing happening for that parameter.

This works just fine:
```
puppet apply -e 'file{"/tmp/test": ensure => file, owner => undef, group => undef, mode => undef}'
```